### PR TITLE
Update Turner.com.xml

### PR DESCRIPTION
**Pull request by TotalCaesar659**
## Type of change

- [x] Existing Ruleset

### CODE REVIEW

**Review by zoracon — CHANGES_REQUESTED** [review]
Errors in test coverage:
```
ERROR src/chrome/content/rules/Turner.com.xml: Duplicated test URL found http://press.turner.com/

ERROR src/chrome/content/rules/Turner.com.xml: Duplicated test URL found http://submission.turner.com/

ERROR src/chrome/content/rules/Turner.com.xml: Duplicated test URL found http://research.turner.com/

ERROR src/chrome/content/rules/Turner.com.xml: Duplicated test URL found http://pressroom.turner.com/

ERROR src/chrome/content/rules/Turner.com.xml: Duplicated test URL found http://sts.turner.com/

ERROR src/chrome/content/rules/Turner.com.xml: Duplicated test URL found http://prensa.turner.com/

ERROR src/chrome/content/rules/Turner.com.xml: Duplicated test URL found http://recruiting.turner.com/

ERROR src/chrome/content/rules/Turner.com.xml: Duplicated test URL found http://news.turner.com/
```

**Comment by TotalCaesar659** [comment]
Hm, strange, I can't find other ruleset with them.

File: src/chrome/content/rules/Turner.com.xml [hunk 1]
```diff
@@ -1,64 +1,190 @@
...
+	<target host="development.turner.com" />
+	<target host="dngupload.turner.com" />
+	<target host="emea.turner.com" />
+	<target host="help.turner.com" />
+	<target host="international.turner.com" />
+	<target host="pm-reporter.iso.turner.com" />
+	<target host="mozart.turner.com" />
+	<target host="music.turner.com" />
+	<target host="news.turner.com" />
+	<target host="prensa.turner.com" />
```
Comment (cschanaj) [inline src/chrome/content/rules/Turner.com.xml:170]: `prensa.turner.com` appeared multiple times in this ruleset, please remove the duplicates within the same file. thanks.

**Review by zoracon — CHANGES_REQUESTED** [review]
Errors with the following domains need attention:
```
ERROR src/chrome/content/rules/Turner.com.xml: Fetch error: http://turner.com/ => https://turner.com/: (7, 'Failed to connect to turner.com port 443: Connection timed out')

ERROR src/chrome/content/rules/Turner.com.xml: Fetch error: http://ttv-content-asperatx.awsbest.turner.com/ => https://ttv-content-asperatx.awsbest.turner.com/: (60, 'SSL certificate problem: certificate has expired')

ERROR src/chrome/content/rules/Turner.com.xml: Fetch error: http://ttv-content-asperatx-all.awsbest.turner.com/ => https://ttv-content-asperatx-all.awsbest.turner.com/: (60, 'SSL certificate problem: certificate has expired')

ERROR src/chrome/content/rules/Turner.com.xml: Fetch error: http://ttv-content-dev-asperatx.dev.awsbest.turner.com/ => https://ttv-content-dev-asperatx.dev.awsbest.turner.com/: (60, 'SSL certificate problem: certificate has expired')

ERROR src/chrome/content/rules/Turner.com.xml: Fetch error: http://ttv-content-dev-asperatx-all.dev.awsbest.turner.com/ => https://ttv-content-dev-asperatx-all.dev.awsbest.turner.com/: (60, 'SSL certificate problem: certificate has expired')

ERROR src/chrome/content/rules/Turner.com.xml: Fetch error: http://zeus.cp-ext.turner.com/ => https://zeus.cp-ext.turner.com/: (6, 'Could not resolve host: zeus.cp-ext.turner.com')

ERROR src/chrome/content/rules/Turner.com.xml: Fetch error: http://help.turner.com/ => https://help.turner.com/: (60, 'SSL certificate problem: certificate has expired')
```

**Pull request closed without merging**